src/SFA.DAS.ApplyService.InternalApi/Infrastructure/InternalQnaApiClient.cs

```diff
@@ -43,6 +43,22 @@ public InternalQnaApiClient(IConfigurationService configurationService, ILogger<
 
             _environmentName = configurationService.GetEnvironmentName();
         }
+
+        public async Task<IEnumerable<ApplicationSequence>> GetSequences(Guid applicationId)
```

Comment:
This method doesn't seem to be used, is it needed when GetSections returns all sections by sequence?

Reply:
I added it for convenience thinking I would need it, however it turns out I don't so have removed it :)